alyx/misc/models.py

```diff
@@ -131,3 +140,145 @@ def save(self):
                     super(Note, self).save()
         else:
             super(Note, self).save()
+
+
+class CageType(BaseModel):
+    description = models.CharField(
+        max_length=1023, blank=True, help_text="Extended description of the cage product/brand")
+
+    def __str__(self):
+        return self.name
+
+
+class Enrichment(BaseModel):
+    name = models.CharField(max_length=255, unique=True,
```

Comment:
No big deal, but for consistency it might be worth using the same trick as in other base models where we override the `name` field (if it works with `unique`)?

```
@modify_fields(name={
    'unique': True,
    'help_text': '...'
})
```